modules/terraform/cmd.go

```diff
@@ -59,3 +65,35 @@ func RunTerraformCommandE(t *testing.T, options *Options, args ...string) (strin
 		return out, retry.FatalError{Underlying: err}
 	})
 }
+
+// GetExitCodeForTerraformCommand runs terraform with the given arguments and options and returns exit code
+func GetExitCodeForTerraformCommand(t *testing.T, additionalOptions *Options, args ...string) int {
+	exitCode, err := GetExitCodeForTerraformCommandE(t, additionalOptions, args...)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return exitCode
+}
+
+// GetExitCodeForTerraformCommandE runs terraform with the given arguments and options and returns exit code
+func GetExitCodeForTerraformCommandE(t *testing.T, additionalOptions *Options, args ...string) (int, error) {
+	options := GetCommonOptions(additionalOptions, args...)
+
+	logger.Log(t, "Running terraform %v", args)
+	cmd := shell.Command{
+		Command:    "terraform",
+		Args:       args,
+		WorkingDir: options.TerraformDir,
+		Env:        options.EnvVars,
+	}
+
+	_, err := shell.RunCommandAndGetOutputE(t, cmd)
+	if err == nil {
+		return TerraformPlanNoChangeExitCode, nil
+	}
+	exitCode, getExitCodeErr := shell.GetExitCodeForRunCommandError(err)
+	if getExitCodeErr == nil {
+		return exitCode, nil
+	}
+	return TerraformPlanExceptionExitCode, getExitCodeErr
```

Comment:
Similarly here, we may want to use `DefaultErrorExitCode`, which is set to 1.

Reply:
done